libs/vkd3d/vkd3d_private.h

```diff
@@ -1496,6 +1521,27 @@ HRESULT vkd3d_create_descriptor_set_layout(struct d3d12_device *device,
         VkDescriptorSetLayoutCreateFlags flags, unsigned int binding_count,
         const VkDescriptorSetLayoutBinding *bindings, VkDescriptorSetLayout *set_layout);
 
+inline const struct d3d12_bind_point_layout *d3d12_root_signature_get_layout(
+        const struct d3d12_root_signature *root_signature, enum vkd3d_pipeline_type pipeline_type)
+{
+    switch (pipeline_type)
+    {
+        case VKD3D_PIPELINE_TYPE_NONE:
```

Comment:
Probably better to break here and let the outer scope return work. This pattern would easily trip warnings due to unreachable code.

Reply:
Fixed.